src/unix/solarish/x86_64.rs

```diff
@@ -1,3 +1,9 @@
+cfg_if! {
+    if #[cfg(target_os = "solaris")] {
+        use xrs_t;
+    }
+}
+
```

Comment:
Could you just use the full path to `xrs_t` where it is used, rather than having a single `cfg`ed import here?

Reply:
Well, not sure what is proper fix. Now I use:
```
cfg_if! {
    if #[cfg(target_os = "solaris")] {
        use unix::solarish::solaris;
    }
}
..
pub uc_xrs: solaris::xrs_t,
```